railties/lib/rails/configuration.rb

```diff
@@ -64,9 +65,10 @@ def unshift(*args, &block)
       end
 
       def merge_into(other) #:nodoc:
-        @operations.each do |operation, args, block|
+        @operations.concat(@delete_operations).each do |operation, args, block|
```

Comment:
Right. I think we only call this once, which is probably ok this way, but what would happen if we call it more times?